CHANGELOG.md

```diff
@@ -3,6 +3,21 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+# 4.3.0 (2020-10-09)
+
+### Components
+- fix(popup-stack): Move adapter var to popup-stack global namespace ([#865](https://github.com/Workday/canvas-kit/pull/865)) [@csongnguyen](https://github.com/csongnguyen)
+- feat(badge): Add canvas-kit-css-badge to canvas-kit-css ([#860](https://github.com/Workday/canvas-kit/pull/860)) [@alanbsmith](https://github.com/alanbsmith)
+- fix(color-picker): Separate popup from color picker ([#846](https://github.com/Workday/canvas-kit/pull/846)) [@mannycarrera4](https://github.com/mannycarrera4)
+- docs(core): Update core CSS border-radius docs ([#871](https://github.com/Workday/canvas-kit/pull/871)) [@alanbsmith](https://github.com/alanbsmith)
+- docs(popup): Change language around use of usePopupStack ([#874](https://github.com/Workday/canvas-kit/pull/874)) [@NicholasBoll](https://github.com/NicholasBoll)
+- feat(popup): Add previousFocusRef to useCloseOnEscape hook ([#873](https://github.com/Workday/canvas-kit/pull/873)) [@alanbsmith](https://github.com/alanbsmith)
+- fix(drawer): Add support for word wrapping in IE11 ([#879](https://github.com/Workday/canvas-kit/pull/879)) [@alanbsmith](https://github.com/alanbsmith)
+- feat(breadcrumbs): Add Breadcrumbs component ([#776](https://github.com/Workday/canvas-kit/pull/776)) [@alanbsmith](https://github.com/alanbsmith)
+
+### Infrastructure
+- Revert "chore: Bump prismjs from 1.17.1 to 1.21.0 ([#826](https://github.com/Workday/canvas-kit/pull/826))" (#870) [@jamesfan](https://github.com/jamesfan)
```

Comment:
This line is a little funky because of the git revert in the PR. Could format it like we do the other PRs:

```suggestion
- fix: Revert prismjs from 1.21.0 back to 1.17.1 ([#870](https://github.com/Workday/canvas-kit/pull/870)) [@jamesfan](https://github.com/jamesfan)
```

Or, if you'd rather preserve the actual commit message, you could just add the proper link to the 870 at the end: `https://github.com/Workday/canvas-kit/pull/870`